lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -2257,37 +2321,30 @@ static bool sinkRescaleQuantizedNode(Function *F) {
       continue;
     }
 
-// Combine Rescale down with Arithmetic node.
-//   ArithmeticNode(Rescale(X), Rescale(Y)) -> ArithmeticNode(X, Y).
-//   ArithmeticNode(Rescale(X), Y) -> ArithmeticNode(X, Y).
-//   ArithmeticNode(X, Rescale(Y)) -> ArithmeticNode(X, Y).
-// Apply this optimization for Add, Sub, Mul, Div, Min, Max.
-#define COMBINE_DOWN_RESCALE_TO_ARITHMETIC_NODE(NODE_NAME_)                    \
```

Comment:
what about MatMul node below, this can be handled the same way.

Reply:
Ah yeah. Changed.